libs/libc/signal/sig_addset.c

```diff
@@ -62,13 +62,12 @@
  *
  ****************************************************************************/
 
-int sigaddset(FAR sigset_t *set, int signo)
+int nxsig_addset(FAR sigset_t *set, int signo)
 {
   /* Verify the signal */
 
   if (!GOOD_SIGNO(signo))
     {
-      set_errno(EINVAL);
       return ERROR;
```

Comment:
Similar to the above.  ERROR should never be returned inside the OS.  It should return -EINVAL if the signal number is invalid.  there are probably changes need in the Return Value comments too.

Reply:
Done.